docs/data/material/components/typography/Types.js

```diff
@@ -5,29 +5,29 @@ import Typography from '@mui/material/Typography';
 export default function Types() {
   return (
     <Box sx={{ width: '100%', maxWidth: 500 }}>
-      <Typography variant="h1" component="div" gutterBottom>
+      <Typography variant="h1" gutterBottom>
```

Comment:
Replicate the tradeoff taken in https://mui.com/joy-ui/react-typography/#levels there are no reasons to have a different tradeoff AFAIK. cc @siriwatknp